Samples/iOS-SwiftUI/iOS-SwiftUI/ContentView.swift

```diff
@@ -200,6 +199,7 @@ struct ContentView: View {
                                 Text("Form Screen")
                             }
                         }
+                        .replayRedact()
```

Comment:
`m`: We already have `.sentryTrace`. Maybe it makes sense also to prefix this with sentry so developers see all sentry related modifies when they type `.sentry` in Xcode.